### PR TITLE
sync 2.5

### PR DESCRIPTION
**Pull request by maxulysse**
Replace and close #718 

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by FriederikeHanssen — APPROVED** [review]

**Comment by FriederikeHanssen** [comment]
Changelog?

**Comment by maxulysse** [comment]
controlfreec tests are working locally, no idea what is actually the issue there...

**Comment by FriederikeHanssen** [comment]
maybe its mpileup it also fails

**Comment by maxulysse** [comment]
Removed the md5sum related to mpileup, I'll see what happens with controlfreec

File: README.md [hunk 1]
```diff
@@ -4,26 +4,24 @@
...
+[![run with singularity](https://img.shields.io/badge/run%20with-singularity-1d355c.svg?labelColor=000000)](https://sylabs.io/docs/)
 [![Launch on Nextflow Tower](https://img.shields.io/badge/Launch%20%F0%9F%9A%80-Nextflow%20Tower-%234256e7)](https://tower.nf/launch?pipeline=https://github.com/nf-core/sarek)
 
-[![Get help on Slack](http://img.shields.io/badge/slack-nf--core%20%23sarek-4A154B?logo=slack)](https://nfcore.slack.com/channels/sarek)
-[![Follow on Twitter](http://img.shields.io/badge/twitter-%40nf__core-1DA1F2?logo=twitter)](https://twitter.com/nf_core)
-[![Watch on YouTube](http://img.shields.io/badge/youtube-nf--core-FF0000?logo=youtube)](https://www.youtube.com/c/nf-core)
+[![Get help on Slack](http://img.shields.io/badge/slack-nf--core%20%23sarek-4A154B?labelColor=000000&logo=slack)](https://nfcore.slack.com/channels/sarek)[![Follow on Twitter](http://img.shields.io/badge/twitter-%40nf__core-1DA1F2?labelColor=000000&logo=twitter)](https://twitter.com/nf_core)[![Watch on YouTube](http://img.shields.io/badge/youtube-nf--core-FF0000?labelColor=000000&logo=youtube)](https://www.youtube.com/c/nf-core)
```
Comment (maxulysse) [inline README.md:16]:
```suggestion
[![Get help on Slack](http://img.shields.io/badge/slack-nf--core%20%23sarek-4A154B?labelColor=000000&logo=slack)](https://nfcore.slack.com/channels/sarek)
[![Follow on Twitter](http://img.shields.io/badge/twitter-%40nf__core-1DA1F2?labelColor=000000&logo=twitter)](https://twitter.com/nf_core)
[![Watch on YouTube](http://img.shields.io/badge/youtube-nf--core-FF0000?labelColor=000000&logo=youtube)](https://www.youtube.com/c/nf-core)
```

File: README.md [hunk 2]
```diff
@@ -68,8 +68,8 @@ By default, the pipeline currently performs the following:
 
 4. Start running your own analysis!
 
-   ```console
-   nextflow run nf-core/sarek -profile <docker/singularity/podman/shifter/charliecloud/conda/institute> --input samplesheet.csv --outdir <OUTDIR> --genome GRCh38
+   ```bash
+   nextflow run nf-core/sarek --input samplesheet.csv --outdir <OUTDIR> --genome GRCh38 -profile <docker/singularity/podman/shifter/charliecloud/conda/institute>
```
Comment (maxulysse) [inline README.md:72]:
```suggestion
   nextflow run nf-core/sarek --input samplesheet.csv --outdir <OUTDIR> --genome GATK.GRCh38 -profile <docker/singularity/podman/shifter/charliecloud/conda/institute>
```

File: docs/usage.md [hunk 3]
```diff
@@ -15,18 +15,18 @@ Sarek is designed to handle single samples, such as single-normal or single-tumo
 
 The typical command for running the pipeline is as follows:
 
-```console
-nextflow run nf-core/sarek --input samplesheet.csv  --outdir <OUTDIR> -profile docker
+```bash
+nextflow run nf-core/sarek --input samplesheet.csv --outdir <OUTDIR> --genome GRCh38 -profile docker
```
Comment (maxulysse) [inline docs/usage.md:19]:
```suggestion
nextflow run nf-core/sarek --input samplesheet.csv --outdir <OUTDIR> --genome GATK.GRCh38 -profile docker
```

File: nextflow.config [hunk 4]
```diff
@@ -146,6 +149,15 @@ profiles {
         shifter.enabled        = false
         charliecloud.enabled   = false
     }
+    mamba {
```
Comment (FriederikeHanssen) [inline nextflow.config:152]: uh cool

**Review by FriederikeHanssen — APPROVED** [review]

**Pull request merged**